sakai-evaluation-tool/src/webapp/content/css/evaluation_base.css

```diff
@@ -2107,3 +2135,7 @@ li.exportIndividual {
 .individualExports {
     text-align: right;
 }
+
+ol.itemListEval span.label {
+    color: #000 !important;
```

Comment:
This was necessary to override default Morpheus styling, which caused the question text while taking an evaluation to be white on a white-ish background, making it impossible to read.

I tried adding more specificity to the selector, but the only way I could get it to apply was by adding the `!important` modifier.